src/main/java/com/arpnetworking/commons/jackson/databind/ImmutableObjectMapper.java

```diff
@@ -167,7 +168,13 @@ public Object intercept(
                 method.setAccessible(true);
                 return null;
             });
-            return method.invoke(_wrapper, args);
+            try {
+                return method.invoke(_wrapper, args);
+            } catch(final InvocationTargetException e){
```

Comment:
Need whitepace after catch and between ) and {. This is in the automated build as a checkstyle violation. You can run these locally through __mvn verify__. Please fix these and we can accept your PR. Thanks!